resources/sip11/ra/pom.xml

```diff
@@ -25,6 +25,24 @@
             <groupId>org.mobicents.servers.jainslee.core</groupId>
             <artifactId>fault-tolerant-ra-api</artifactId>			
         </dependency>
+		<!-- For testing purposes -->
+		<dependency>
+			<groupId>org.mockito</groupId>
```

Comment:
we have both Restcomm level poms (https://github.com/RestComm/parent/blob/2.x/pom.xml), and per project level poms https://github.com/RestComm/jain-slee.sip/blob/master/pom.xml). We may evaluate to move these dependencies to a higher scope...